package.json

```diff
@@ -90,7 +90,9 @@
     "typescript": "^5.1.3"
   },
   "resolutions": {
-    "tar": "6.2.1"
+    "tar": "6.2.1",
+    "@sentry/react": "^7.19.1",
```

Comment:
@YounixM 
on the signoz repo we have - `"@sentry/react": "8.41.0",` and also `^` does not cater -upgrade to 8.x.x, since major version changes often introduce breaking changes. 

approving if this is expected and according to the other dependencies and compatibility of this repo

Reply:
To Fix: https://github.com/SigNoz/signoz-web/security/dependabot/17